### DISCUSSION TITLE
onnx模型转float16后性能异常

### DISCUSSION THREAD

**Discussion by szwlh-c** [PaddlePaddle/PaddleOCR#13650] [Q&A]
我用onnxconverter_common.float16.convert_float_to_float16把ch_PP-OCRv3的onnx转成float16，用onnxruntime推理时会报以下警告。
![image](https://github.com/user-attachments/assets/045ff215-0d83-4d5e-9d2d-ac83fb0fb1ef)
速度也明显变慢，请问是什么原因呢？（v4的模型试了也有同样的问题）

**Comment by szwlh-c** [comment]
可以定位到是cllip的问题
![image](https://github.com/user-attachments/assets/7dcb36ea-7684-4f53-b9d6-de7bbc812aba)
![image](https://github.com/user-attachments/assets/7a301f62-d3a4-46fb-b5a8-0b798ec38381)

**Comment by GreatV** [comment]
建议去paddle2onnx提问
